tools/ui-components/src/tab-pane/tab-pane.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+import { TabPaneProps } from './types';
+
+const defaultClasses =
+  'block relative mr-1 border-1 border-solid border-background-tertiary border-b-transparent';
```

Comment:
I haven't added `line-height` and `cursor`, because our usage, we are only displaying this for touch user, doesn't include these value, and they will take compute power for no reason